tests/per_provider/provider_cfg/tpm/config.toml

```diff
@@ -16,4 +16,4 @@ manager_type = "OnDisk"
 provider_type = "Tpm"
 key_id_manager = "on-disk-manager"
 tcti = "mssim"
-owner_hierarchy_auth = "tpm_pass"
+owner_hierarchy_auth = ""
```

Comment:
Is that not good anymore?

Reply:
😅 my mistake, I changed that for local testing, but forgot to reset it